From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Tue, 14 Aug 2012 19:45:18 +0300
Subject: Fix libbz2.so version numbers

Fixes the .so version number for the suse-bzip2. Otherwise we may get an
error from the dynamic linker when trying to run bzip2 from suse-bzip2.

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>
---
 pit/suse-bzip2/Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pit/suse-bzip2/Makefile b/pit/suse-bzip2/Makefile
index 4960037..0977534 100644
--- a/pit/suse-bzip2/Makefile
+++ b/pit/suse-bzip2/Makefile
@@ -12,8 +12,8 @@
 # in the file LICENSE.
 # ------------------------------------------------------------------
 
-somajor=1.0
-sominor=$(somajor).4
+somajor=1
+sominor=$(somajor).0.6
 SHELL=/bin/sh
 
 # To assist in cross-compiling
